### ISSUE TITLE
Exit trap is wrongfully triggered when `--help` is issued

### ISSUE THREAD

**Issue by Lin-Buo-Ren** [WoeUSB/WoeUSB#19]
## Describe the bug ##

As title

## To Reproduce ##

Run `./path/to/woeusb --help`

## Expected behavior ##

Only help text is printed

## Teminal output ##

If applicable, add the terminal output to help explain your problem.  Please avoid pasting screenshots whenever possible.

```your-woeusb-output-here:
WoeUSB 5.0.1 Help Information
======================================

...stripped

## Command-line Options ##
    --verbose, -v
        Verbose mode

...stripped

    --for-gui
        No longer supported, reserved for compatibility with the wrapper programs

Unmounting and removing "/tmp/woeusb-source-20201116-065010-Monday.V7gxBi"...
umount: /tmp/woeusb-source-20201116-065010-Monday.V7gxBi: not mounted.
cleanup_mountpoint: Warning: Unable to unmount "/tmp/woeusb-source-20201116-065010-Monday.V7gxBi".
Unmounting and removing "/tmp/woeusb-target-20201116-065010-Monday.KKSQG0"...
umount: /tmp/woeusb-target-20201116-065010-Monday.KKSQG0: not mounted.
cleanup_mountpoint: Warning: Unable to unmount "/tmp/woeusb-target-20201116-065010-Monday.KKSQG0".
Some mountpoints are not unmount/cleaned successfully and must be done manually
```

## Environment ##

- OS: Ubuntu
- Bash: 5.0.17
- WoeUSB: 5.0.1

**Comment by koraysevinc** [comment]
This issue still exists.

**Comment by Lin-Buo-Ren** [comment]
This bug is fixed in WoeUSB 5.0.8.
